containers-template/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { Container, loadBalance, getContainer } from "@cloudflare/containers";
+
+export class MyContainer extends Container {
+  defaultPort = 8080;
+  sleepAfter = "2m";
+  envVars = {
+    MESSAGE: "I was passed in via the container class!",
+  };
+
+  override onStart() {
+    console.log("Container successfully started");
+  }
+
+  override onStop() {
+    console.log("Container successfully shut down");
+  }
+
+  override onError(error: unknown) {
+    console.log("Container error:", error);
+  }
+}
+
+export default {
+  async fetch(
+    request: Request,
+    env: { MY_CONTAINER: DurableObjectNamespace<MyContainer> },
+  ): Promise<Response> {
+    const pathname = new URL(request.url).pathname;
+
+    // To route requests to a specific container,
+    // pass a unique container identifier to .get()
+    if (pathname.startsWith("/container")) {
```

Comment:
I would consider converting this routing logic to a Hono router. I know this is a hello-world example, but a router is needed to build anything meaningful. New users who use this may not know a router exists and end up going way too far with this style of routing and feeling like Workers is difficult to use.

Defaults make a big difference in user experience

Reply:
+1 to this. The initial simplicity of path-based routing quickly turns into a stumbling block for new users. We are in the process of adding Hono as a requirement to the Contributor Guidelines.